src/shared-components/Tooltip/Tooltip.spec.tsx

```
@@ -0,0 +1,270 @@
+
+    await waitFor(() => {
+      expect(tooltip).toHaveAttribute('aria-hidden', 'true');
+    });
+  });
+
+  test('shows tooltip on focus', async () => {
+    const user = userEvent.setup({ advanceTimers: vi.advanceTimersByTime });
+
+    render(
+      <Tooltip content="Tooltip text" delayShow={0}>
+        <button>Focus me</button>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)